cppQML/.vscode/tasks.json

```diff
@@ -120,10 +137,12 @@
             ],
             "dependsOrder": "sequence",
             "dependsOn": [
+                "kill-qt-creator",
                 "update-qt-ini-arm64",
                 "kill-redirect-qml-debugger",
                 "deploy-torizon-arm64",
-                "redirect-qml-debugger"
+                "redirect-qml-debugger",
+                "open-in-qt-creator"
```

Comment:
Isn't it better to putt the the `open-in-qt-creator` before the `deploy-torizon-arm64`, because it will take much less time to reopen, it will look much more like a "reload-window"

Reply:
Or like this, as `deploy-torizon-arm64` validates that the arch is correct:
                "kill-redirect-qml-debugger",
                "deploy-torizon-arm64",
                 "kill-qt-creator",
                "update-qt-ini-arm64",
                "open-in-qt-creator",
                "redirect-qml-debugger"